state-chain/pallets/cf-validator/src/lib.rs

```diff
@@ -1399,10 +1398,11 @@ impl<T: Config> pallet_session::SessionManager<ValidatorIdOf<T>> for Pallet<T> {
 	/// These Validators' keys must be registered as part of the session pallet genesis.
 	fn new_session_genesis(_new_index: SessionIndex) -> Option<Vec<ValidatorIdOf<T>>> {
 		let genesis_authorities = Self::current_authorities();
-		assert!(
-			!genesis_authorities.is_empty(),
-			"No genesis authorities found! Make sure the Validator pallet is initialised before the Session pallet."
-		);
+		if !genesis_authorities.is_empty() {
```

Comment:
Hmmm, better to return None here? not sure we want to start a session with no validators...